azure-pipelines.yml

```diff
@@ -71,5 +75,22 @@ jobs:
         artifactName: 'pros_cli-$(Build.BuildNumber)-linux-$(buildArch)'
         targetPath: 'out'
 
-
+- job: PublishArtifact
+  dependsOn:
+    - UpdateBuildNumber
+    - Linux
+  condition: and(succeeded(), or(eq(variables['Build.SourceBranch'], 'refs/heads/master'), eq(variables['Build.SourceBranch'], 'refs/heads/develop')))
+  steps:
+    - checkout: none
+    - task: DownloadPipelineArtifact@0
+      inputs:
+        artifactName: 'pros_cli-$(Build.BuildNumber)-linux-x64'
+        targetPath: $(System.DefaultWorkingDirectory)
+    - script: 'sudo pip install twine'
+    - task: TwineAuthenticate@0
+      inputs:
+        artifactFeeds: 'pros'
```

Comment:
Can you rename this feed to `pros-cli`? It's a little more explicit and we may want to do something with the kernel on azure in the future